### ISSUE TITLE
research: silent TTS output when moshi_tokens=zeros forced in tts_stream

### ISSUE THREAD

**Issue by displague** [displague/agent-bot#4]
## Background

When implementing teacher-forced TTS (generating speech from text tokens without user audio input), the initial approach passed an explicit \moshi_tokens=zero_frame\ to force the output audio codebook tokens to zero/silence, reasoning that the depformer should still decode something meaningful.

## What Was Tried

\\\python
# In tts_stream(), per-frame step call:
result = self.lm_gen.step(
    text_token=tok,
    input_tokens=sine_frame,   # 1920-sample sine wave as user audio
    moshi_tokens=torch.zeros((1, dep_q, 1), dtype=torch.long, device=device)
)
\\\

## Failure Mode

All output frames were silent (amplitude ~0.001, near-zero). Decoding showed only PAD token indices. The forced zero \moshi_tokens\ replaced the depformer's autoregressive sampling — the model had no ability to generate real audio output because the output tokens were being overridden with padding.

## Resolution (Closed)

Omit \moshi_tokens\ entirely. Without this argument, \udio_provided=False\ in the LM step — the depformer samples audio tokens autoregressively, conditioned on the text token stream. Real speech amplitude (0.19–0.55) appears from frame 3 onward (first 1-2 frames are residual silence from system prompt pipeline context).

\\\python
result = self.lm_gen.step(
    text_token=tok,
    input_tokens=sine_frame,  # user audio (sine avoids VAD silence detection)
    # NO moshi_tokens — let depformer sample freely
)
\\\

## Key Insight

Moshi is a full-duplex model — \moshi_tokens\ is the **output** codebook (agent speech), not input. Providing it forces the output, defeating the purpose. The model only speaks when given conversational context (silence → generates silence; text token → generates speech).
